src/app/components/JumpTo/index.stories.tsx

```diff
@@ -23,5 +31,14 @@ export const Example = () => {
       block => block.type === 'jumpTo',
     )?.model.jumpToHeadings;
 
-  return <Component jumpToHeadings={jumpToHeadings} />;
+  const showRelatedContentLink =
+    !!pidginArticleFixtureWithJumpToBlock.data.article.content.model.blocks.find(
+      block => block.type === 'relatedContent',
+    );
+  return (
+    <Component
+      jumpToHeadings={jumpToHeadings}
+      showRelatedContentLink={showRelatedContentLink}
+    />
+  );
 };
```

Comment:
Worth deleting the `blocks.find` stuff and just having a separate story for the related content link?

Like:
```
 <Component
      jumpToHeadings={jumpToHeadings}
      showRelatedContentLink
    />
```

Reply:
Finally i have managed to add this!